docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.4'
+
+services:
+  app:
+    build:
+      context: .
+      target: development
+    image: bbb-stress-test:development
+    working_dir: /app/
+    user: ${DOCKER_USER:-1000}
+    cap_add:
```

Comment:
Didn't know about this :clap:

Reply:
Thanks to [this documentation](https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-in-docker) :slightly_smiling_face: